src/components/VaultSummary.tsx

```diff
@@ -2,50 +2,56 @@ import { useMemo } from 'react';
 import { useVaultStore } from 'store/vaults';
 import { useAtomValue } from 'jotai';
 import { displayFunctionsAtom } from 'store/app';
-import type { VaultKey } from 'store/vaults';
+import SkeletonVaultSummary from 'components/SkeletonVaultSummary';
 
 type Props = {
-  vaultKey: VaultKey;
+  offerId: string;
 };
 
-const VaultSummary = ({ vaultKey }: Props) => {
+const VaultSummary = ({ offerId }: Props) => {
```

Comment:
this is the offerId in which the vault was acquired? would help to document that as there are offerIds for changing and closed vaults too.

but I wonder about changing this to key off that. seems like a lower level implementation detail of how we get the vault data. Can't the `VaultStore` continue to key by `vaultKey`?

Reply:
I think it's useful to know we are looking at the offerId. When we implement adjust/close offers we'll need to pass in an offerId for the continuing invitation, so we'll need to store it regardless. The VaultKey thing was from when it didn't have access to an offerId and was showing everyone's vaults at once.

Reply:
> When we implement adjust/close offers we'll need to pass in an offerId for the continuing invitation, so we'll need to store it regardless

Yes we'll have to store the mapping of vaultId with the offerId that created the continuing invitations, but the UI code should not be concerned with that. It's only concerned with what vault it's displaying and operating on. The underlying stores and services can manage turning that into whatever request format is necessary to accomplish the goal.

Reply:
Okay changed to `vaultKey` and added `createdByOfferId` as a separate property on the vault.